### PR TITLE
[FEAT] Dev script

### PR DESCRIPTION
**Pull request by hv7214**
**Details**
Used `npm-run-all` package to run `serve` and `fakeapi` script in parallel.

**Note**
1. `npm-run-all` is just a package, it's nothing to do with `npm`. 
2. Currently no guidelines are added regarding this in `README.md`. 

**Related issue**
https://github.com/moira-alert/moira/issues/273

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,12 +21,13 @@
         "jest": "24.5.0",
         "lodash": "4.17.13",
         "moment": "2.24.0",
+        "npm-run-all": "^4.1.5",
```
Comment (sashasushko) [inline package.json]: Please, install exact version. (`-E` option)

File: package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
         "build": "cross-env NODE_ENV=production webpack --config webpack.prod.js",
         "storybook": "cross-env NODE_ENV=development start-storybook -p 9001 -c .storybook --ci",
         "fakeapi": "json-server --watch fakeapi/db.json --routes fakeapi/routes.json --port 9002",
+        "dev": "npm-run-all --parallel start fakeapi",
```
Comment (sashasushko) [inline package.json]: Let's some refactoring:
1. Rename `start` → `serve` and `start-local-api` → `serve-with-local-api`
2. Then rename `dev` → `start`
3. Fix npm-run-all command by point 1

So, at end we'll get short `yarn start` command for starting local developing

File: package.json (hunk 1 above)
Comment (hv7214) [reply package.json]: okay sure

File: package.json (hunk 2 above)
Comment (hv7214) [reply package.json]: yeah it's will be great to refactor like this.

File: package.json [hunk 3]
```diff
@@ -3,11 +3,12 @@
     "version": "1.1.0",
     "license": "MIT",
     "scripts": {
-        "start": "cross-env NODE_ENV=development webpack-dev-server --config webpack.dev.js",
-        "start-local-api": "cross-env API_MODE=local yarn start",
+        "serve": "cross-env NODE_ENV=development webpack-dev-server --config webpack.dev.js",
+        "start-with-local-api": "cross-env API_MODE=local yarn start",
```
Comment (sashasushko) [inline package.json]:
```suggestion
        "start-with-local-api": "cross-env API_MODE=local yarn serve",
```

Comment (sashasushko) [reply package.json]: It's no need to parallel scripts for local api

**Pull request merged**